instrumentation/reactor/reactor-netty/reactor-netty-0.9/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/reactornetty/v0_9/ReactorNettyHttpClientTest.java

```diff
@@ -38,7 +39,17 @@ HttpClient createHttpClient(boolean readTimeout) {
   protected void configure(HttpClientTestOptions.Builder optionsBuilder) {
     super.configure(optionsBuilder);
 
-    optionsBuilder.setSingleConnectionFactory(ReactorNettyHttpClientTest::createSingleConnection);
+    boolean isWindows = OS.WINDOWS.isCurrentOs();
+
+    // Disable remote connection tests on Windows due to reactor-netty creating extra spans
+    if (isWindows) {
+      optionsBuilder.setTestRemoteConnection(false);
+    }
+
+    // Only run single connection tests on Linux due to networking stack differences
+    if (!isWindows) {
+      optionsBuilder.setSingleConnectionFactory(ReactorNettyHttpClientTest::createSingleConnection);
+    }
```

Comment:
not sure if you were keeping these conditions separate intentionally for readability, but if not, could simplify

```suggestion
    if (OS.WINDOWS.isCurrentOs()) {
      // Disable remote connection tests on Windows due to reactor-netty creating extra spans
      optionsBuilder.setTestRemoteConnection(false);
    } else {
      // Only run single connection tests on Linux due to networking stack differences
      optionsBuilder.setSingleConnectionFactory(ReactorNettyHttpClientTest::createSingleConnection);
    }
```